docs/src/concepts.md

```diff
@@ -0,0 +1,160 @@
+# Concepts
+
+## Multiphoton Calcium Imaging
+Over the past two decades, in vivo two-photon laser-scanning imaging of calcium signals has evolved into a mainstream modality for neurophysiology experiments to record population activity in intact neural circuits. The tools for signal acquisition and analysis continue to evolve but common patterns and elements of standardization have emerged.
+
+The preprocessing workflow for two-photon laser-scanning microscopy includes motion correction (rigid or non-rigid), cell segmentation, and calcium event extraction (sometimes described as "deconvolution" or "spike inference"). Some include raster artifact correction, cropping and stitching operations.
+
+<figure markdown>
+  ![Raw Scans](./images/rawscans.gif){: style="height:250px"}
+  ![Motion Corrected Scans](./images/motioncorrectedscans.gif){: style="height:250px"}
+  ![Cell Segmentation](./images/cellsegmentation.png){: style="height:250px"}
+  ![Calcium Events](./images/calciumtraces.png){: style="height:250px"}
+  <figcaption>Left to right: Raw scans, Motion corrected scans, Cell segmentation, Calcium events</figcaption>
+</figure>
+
+For a long time, most labs developed custom processing pipelines, sharing them with others as academic open-source projects. This has changed recently with the emerging of a few leaders as the standardization candidates for the initial preprocessing.
+
+- [CaImAn](https://github.com/flatironinstitute/CaImAn) (Originally developed by Andrea Giovannucci, current support by FlatIron Institute: Eftychios A. Pnevmatikakis, Johannes Friedrich)
+- [Suite2p](https://github.com/MouseLand/suite2p) (Carsen Stringer and Marius Pachitariu at Janelia), 200+ users, active support
+
+Element Calcium Imaging encapsulates these packages to ease the management of data and its analysis.
+
+## Key partnerships
+Over the past few years, several labs have developed DataJoint-based data management and processing pipelines for two-photon Calcium imaging. Our team collaborated with several of them during their projects. Additionally, we interviewed these teams to understand their experiment workflow, pipeline design, associated tools, and interfaces.
+
+These teams include:
+
++ MICrONS (Andreas Tolias Lab, BCM) - https://github.com/cajal
++ BrainCoGs (Princeton) - https://github.com/BrainCOGS
++ Moser Group (Kavli Institute/NTNU) - private repository
++ Anne Churchland Lab (UCLA)
+
+
+## Acquisition tools
+
+### Hardware
+The primary acquisition systems are:
+
++ Sutter (we estimate 400 rigs in active use - TBC)
```

Comment:
What's `TBC`?

Reply:
@kabilar would be the best person to answer this.

Reply:
Not sure.  Let's remove.